addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1925,6 +1925,15 @@ options.registry.HeaderNavbar = options.Class.extend({
         if (widgetName === 'option_logo_height_scrolled') {
             return !!this.$('.navbar-brand').length;
         }
+        if (widgetName === 'header_alignment_opt') {
```

Comment:
Just checking the commit message, second point "- The "right/left" alignment option was reversed with the "vertical"
header template." -> I wanted to test, but it seems to be always centered no matter what. Maybe it's another case where it is reversed? :thinking:

Reply:
Actually the "aligment" options only impacts the mobile menu and the issue was only with "off-canvas".

Reply:
Strange I am sure it impacted the desktop alignment at a time. The label was actually "Alignment (+ 📱)" to indicate it impacts desktop and if not, at least mobile. At a time, FP/SBU/PDE did not like that phone icon and we removed it... but if the "Alignment" option started to impact mobile only... why did not we rename it "Mobile Alignment"? :grimacing: 

Maybe we could to do that too in this PR: if some are impacted in desktop, in `updateUI` change the label to something like "Mobile Alignment" for those who are not impacted in desktop?

Reply:
> but if the "Alignment" option started to impact mobile only... why did not we rename it "Mobile Alignment"? 

Were you able to confirm that was not a bug and really a choice (that some header were only impacted in mobile by the "Alignment" option)? In any case, even if it's a bug, I guess that we'll keep it as it is in a stable version but... not if we broke it last week. And could you link (in the commit message) the commit that made those changes so I can properly review? Thank you :+1:

Reply:
> > but if the "Alignment" option started to impact mobile only... why did not we rename it "Mobile Alignment"?
> 
> Were you able to confirm that was not a bug and really a choice (that some header were only impacted in mobile by the "Alignment" option)? In any case, even if it's a bug, I guess that we'll keep it as it is in a stable version but... not if we broke it last week. And could you link (in the commit message) the commit that made those changes so I can properly review? Thank you +1


It was actually called "hamburger position" at the beginning : [see here](https://github.com/odoo/odoo/commit/2a1aa808e939eeaa3caec6a1a82e19f023f1d010#diff-9f41c68ba854940dda3361dc32795e2d7e9fc5b83ef7f8afd35a455b493e9930L590-L594)